src/components/ContentTypesPanel/Categories/Categories.js

```diff
@@ -36,7 +36,7 @@ class Categories extends Component {
         console.log('Error: ', error);
       });
   };
-
+//NewHook
   static getDerivedStateFromProps = (nextProps, prevState) => {
```

Comment:
getDerivedStateFromProps doesn't use 'nextprops' and 'prevstate' as arguments it is using 'props' and 'state'
;-)